django_ses/urls.py

```diff
@@ -1,8 +1,8 @@
-from django.conf.urls import url
+from django.conf.urls import path
```

Comment:
Hi. I believe the correct namespace path is `django.urls` 
Then the import should be `from django.urls import path`

Thank you for your hard work on this library.

Reply:
I'm getting the following exception after this release: 
```
Exception in thread django-main-thread:
Traceback (most recent call last):
  File "/usr/lib/python3.8/threading.py", line 932, in _bootstrap_inner
    self.run()
  File "/usr/lib/python3.8/threading.py", line 870, in run
    self._target(*self._args, **self._kwargs)
  File "/home/filipe/.local/share/virtualenvs/inBuildBE-kkJCOQop/lib/python3.8/site-packages/django/utils/autoreload.py", line 64, in wrapper
    fn(*args, **kwargs)
  File "/home/filipe/.local/share/virtualenvs/inBuildBE-kkJCOQop/lib/python3.8/site-packages/django/core/management/commands/runserver.py", line 118, in inner_run
    self.check(display_num_errors=True)
  File "/home/filipe/.local/share/virtualenvs/inBuildBE-kkJCOQop/lib/python3.8/site-packages/django/core/management/base.py", line 419, in check
    all_issues = checks.run_checks(
  File "/home/filipe/.local/share/virtualenvs/inBuildBE-kkJCOQop/lib/python3.8/site-packages/django/core/checks/registry.py", line 76, in run_checks
    new_errors = check(app_configs=app_configs, databases=databases)
  File "/home/filipe/.local/share/virtualenvs/inBuildBE-kkJCOQop/lib/python3.8/site-packages/django/core/checks/urls.py", line 13, in check_url_config
    return check_resolver(resolver)
  File "/home/filipe/.local/share/virtualenvs/inBuildBE-kkJCOQop/lib/python3.8/site-packages/django/core/checks/urls.py", line 23, in check_resolver
    return check_method()
  File "/home/filipe/.local/share/virtualenvs/inBuildBE-kkJCOQop/lib/python3.8/site-packages/django/urls/resolvers.py", line 412, in check
    for pattern in self.url_patterns:
  File "/home/filipe/.local/share/virtualenvs/inBuildBE-kkJCOQop/lib/python3.8/site-packages/django/utils/functional.py", line 48, in __get__
    res = instance.__dict__[self.name] = self.func(instance)
  File "/home/filipe/.local/share/virtualenvs/inBuildBE-kkJCOQop/lib/python3.8/site-packages/django/urls/resolvers.py", line 598, in url_patterns
    patterns = getattr(self.urlconf_module, "urlpatterns", self.urlconf_module)
  File "/home/filipe/.local/share/virtualenvs/inBuildBE-kkJCOQop/lib/python3.8/site-packages/django/utils/functional.py", line 48, in __get__
    res = instance.__dict__[self.name] = self.func(instance)
  File "/home/filipe/.local/share/virtualenvs/inBuildBE-kkJCOQop/lib/python3.8/site-packages/django/urls/resolvers.py", line 591, in urlconf_module
    return import_module(self.urlconf_name)
  File "/usr/lib/python3.8/importlib/__init__.py", line 127, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 1014, in _gcd_import
  File "<frozen importlib._bootstrap>", line 991, in _find_and_load
  File "<frozen importlib._bootstrap>", line 975, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 671, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 848, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/home/filipe/bairesdev/inBuildBE/inBuildBE/urls.py", line 66, in <module>
    path("admin/django-ses/", include("django_ses.urls")),
  File "/home/filipe/.local/share/virtualenvs/inBuildBE-kkJCOQop/lib/python3.8/site-packages/django/urls/conf.py", line 34, in include
    urlconf_module = import_module(urlconf_module)
  File "/usr/lib/python3.8/importlib/__init__.py", line 127, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 1014, in _gcd_import
  File "<frozen importlib._bootstrap>", line 991, in _find_and_load
  File "<frozen importlib._bootstrap>", line 975, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 671, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 848, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/home/filipe/.local/share/virtualenvs/inBuildBE-kkJCOQop/lib/python3.8/site-packages/django_ses/urls.py", line 1, in <module>
    from django.conf.urls import path
ImportError: cannot import name 'path' from 'django.conf.urls' (/home/filipe/.local/share/virtualenvs/inBuildBE-kkJCOQop/lib/python3.8/site-packages/django/conf/urls/__init__.py)

```

It resumes working after changing the import line.

Reply:
Thanks! Would you open a PR with the fix?

Reply:
I ended up pushing a fix 8bfd17fed03480453ec5756009abd804135dac99

Reply:
Try 2.1.1 https://pypi.org/project/django-ses/2.1.1/